Tables/CMIP6_Emon.json

```diff
@@ -2232,10 +2232,10 @@
             "frequency": "mon", 
             "modeling_realm": "atmos", 
             "standard_name": "eastward_atmosphere_dry_static_energy_transport_across_unit_distance", 
-            "units": "1.e6 J m-1 s-1", 
+            "units": "MJ m-1 s-1", 
```

Comment:
Are we positive `udunits` can handle `MJ`?

Reply:
It should be able to according to the documentation.